docs/appendices/release-notes/5.9.5.rst

```diff
@@ -1,17 +1,12 @@
 .. _version_5.9.5:
 
-==========================
-Version 5.9.5 - Unreleased
-==========================
+=============
+Version 5.9.5
+=============
 
 
-.. comment 1. Remove the " - Unreleased" from the header above and adjust the ==
-.. comment 2. Remove the NOTE below and replace with: "Released on 20XX-XX-XX."
-.. comment    (without a NOTE entry, simply starting from col 1 of the line)
 .. NOTE::
-
-    In development. 5.9.5 isn't released yet. These are the release notes for
-    the upcoming release.
+    Released on 2024-12-10
```

Comment:
```suggestion
    Released on 2024-12-10.
```

Reply:
The text is usually also above the `::NOTE`